system/settings/Kconfig

```diff
@@ -0,0 +1,76 @@
+#
+# Settings Storage Configuration
+#
+
+config SYSTEM_SETTINGS
+  bool "Settings Functions"
```

Comment:
tab

Reply:
Every time a set VSCode up to not convert tabs to spaces for Kconfig files, something (VS Code updates maybe) comes along and that setting gets lost. Fixed now, ready for my next commit.